### ISSUE TITLE
Raspberry gstreamer broken

### ISSUE THREAD

**Issue by fnoop** [fnoop/maverick#242]
Compiled gstreamer failing visiond on boot:
```
Jan  7 08:17:45 fnoop-raspberry systemd[1]: maverick-visiond.service: main process exited, code=exited, status=1/FAILURE
Jan  7 08:17:46 fnoop-raspberry node[728]: Starting standalone
Jan  7 08:17:46 fnoop-raspberry maverick-visiond[866]: ** (process:866): WARNING **: Failed to load shared library 'libgstreamer-1.0.so.0' referenced by the typelib: libgstreamer-1.0.so.0: cannot open shared object file: No such file or directory
Jan  7 08:17:46 fnoop-raspberry maverick-visiond[866]: Traceback (most recent call last):
Jan  7 08:17:46 fnoop-raspberry maverick-visiond[866]: File "/srv/maverick/software/maverick/bin/maverick-visiond", line 19, in <module>
Jan  7 08:17:46 fnoop-raspberry maverick-visiond[866]: Gst.init(None)
Jan  7 08:17:46 fnoop-raspberry maverick-visiond[866]: GLib.Error: g-invoke-error-quark: Could not locate gst_init: libgstreamer-1.0.so.0: cannot open shared object file: No such file or directory (1)
Jan  7 08:17:46 fnoop-raspberry systemd[1]: maverick-visiond.service: control process exited, code=exited status=1
Jan  7 08:17:46 fnoop-raspberry systemd[1]: Unit maverick-visiond.service entered failed state.
```

Also seems to reboot raspberry randomly.

**Comment by fnoop** [comment]
omxh264enc broken.  Capturing x264 and passing straight through to rtp works fine.

**Comment by fnoop** [comment]
gst-plugins-bad compile broken.

**Comment by fnoop** [comment]
Fixed vision systemd startup, needed to add environment to service manifests.

**Comment by fnoop** [comment]
Update to 1.10.3

**Comment by fnoop** [comment]
gst-omx 1.10.3 compiles but hangs, like 1.10.2
https://github.com/RPi-Distro/repo/issues/43

**Comment by fnoop** [comment]
Try follow exact instructions from:
https://lists.freedesktop.org/archives/gstreamer-openmax/2013-March/000724.html

**Comment by fnoop** [comment]
So, gst-omx for raspberry (collabora) has dropped maintaining patches against 1.0.0.1.   Old patched branch no longer compiles against 1.10 or newer gstreamer installs, and newer gst-omx branches don't work for raspberry.  So for now, no option but to revert to raspbian binary install for gstreamer.
